bokeh/io.py

```diff
@@ -576,12 +576,14 @@ def _wait_until_render_complete(driver):
       window._bokeh_render_complete = true;
     }
 
-    var doc = window.Bokeh.documents[0];
+    if (window.Bokeh) {
```

Comment:
Some of the server apps load slowly, so sometimes the page is loaded before Bokeh is appended to the window. There's probably a better way to handle this.